### ISSUE TITLE
Numpy v1.14.0 not supported.

### ISSUE THREAD

**Issue by jesuscast** [tensorly/tensorly#26]
As of Jan 12, the latest version of Numpy is not supported.
Running:
`TENSORLY_BACKEND=numpy pytest -v --cov tensorly tensorly`

The tests fail inside `test_kronecker()` at:
```python
 for i, shape in enumerate(shapes):
    T.assert_array_equal(res, kr)
```

**Comment by JeanKossaifi** [comment]
Thanks for noticing and reporting the issue, I'll check what is going on!
I would prefer the package to be compatible with any recent version of NumPy and Scipy rather than impose a specific version in the requirements..

**Comment by JeanKossaifi** [comment]
Could you post more details about your error? Are you using Python 3?

I just tested it in a separate environment with NumPy 1.4.0 and had no issue.

**Comment by JeanKossaifi** [comment]
Fixed by 3dbf6bdc8595a8fdc8cf5f3c08178f9c4b223e2f
